packages/patternfly-4/react-core/src/components/Switch/examples/Switch.md

```diff
@@ -6,9 +6,63 @@ propComponents: ['Switch']
 typescript: true
 ---
 
-import { Switch } from '@patternfly/react-core';
+import { Switch, Table, TableHeader, TableBody, Button } from '@patternfly/react-core';
 
 ## Examples
+```js title=Multi switch table
```

Comment:
I don't think we should add this example of the switch here.  It seems to be a specific use case.  If you want to verify it works the demo-app and integration test would be a better place for this.

Reply:
good idea

Reply:
Use dashes between words as well, i.e. `title=Multi-switch-table`